source/templates/classic/style.css

```diff
@@ -1,4 +1,5 @@
 /* SVG global context */
+  @import 'node_modules/modern-normalize/modern-normalize.css';
```

Comment:
Sadly this doesn't seems to work, chrome still output oversized twemojis

Reply:
Yup. What a pain. I tried changing CSS, height & width things, viewbox. Nothing.

:(

Reply:
I found a solution in a0c70b1, though it seems a bit magic

Reply:
Whatever works works!